content/docs/command-reference/add.md

```diff
@@ -7,7 +7,9 @@ file.
 
 ```usage
 usage: dvc add [-h] [-q | -v] [-R] [--no-commit] [--external]
-               [--file <filename>] [--glob] [--desc <text>]
+               [--glob] [--file <filename>] [-o <path>]
+               [--to-remote] [-r <name>] [-j <number>]
+               [--desc <text>]
```

Comment:
`-r` will be merged into `--to-remote` according to https://github.com/iterative/dvc/pull/5198#discussion_r556712627, so:
```suggestion
               [--to-remote] [-j <number>] [--desc <text>]
```

Reply:
Those are separate right now, so let's keep them separate in the docs for now.

Reply:
`-r name` also makes sense in a feature of being able to specify default remote for a specific data, so it makes sense to keep `-r name` here.

Reply:
OK. We did agree to merge them right? In https://github.com/iterative/dvc/pull/5198#pullrequestreview-566902933

Is there an issue to followup on those details? Thanks

p.s. the ideas is that `--to-remote` can accept the remote name as (optional) argument @efiop